src/packs/abilities/Censor_JY7vFbefcnBYK2ly/Level_5_cPfWKGS4u0dYaVqr/9_Wrath_uz338MRS29LehXPl/ability_Orison_of_Victory_y0aUnANMcvPv1bW6.json

```
@@ -44,7 +44,7 @@
           "_id": "jse7yEixzHl7s2NY",
           "other": {
             "tier1": {
-              "display": "Each target gains 1 surge."
+              "display": "Each target gains 1 surges."
```

```suggestion
              "display": "Each target gains 1 surge."
```
Keep the source text even if our enricher always pluralizes